CONTRIBUTING.md

```diff
@@ -2,9 +2,36 @@
 
 Hi! Thanks for your interest in helping make
 [WebExtension](https://developer.mozilla.org/en-US/Add-ons/WebExtensions)
-development more awesome by contributing to the `web-ext` tool.
-
-#Picking an issue
+development more awesome by contributing to the `web-ext` tool. Here are links to all the sections in this document:
+
```

Comment:
You should put a comment here that says how to run `doctoc`. Also, you'll need to add it to `package.json` as a dev requirement and make a script we can use like `npm run gen-contributing-toc`.